Makefile

```diff
@@ -40,5 +40,8 @@ Gopkg.lock: Gopkg.toml
 Gopkg.toml: $(SOURCES)
 	@if ! dep check; then touch $@; fi;
 
-fmt:
-	gofmt -w $(SRC_DIRS)
+fmt: vendor
+	goreturns -w pkg/ cmd/
```

Comment:
Are you able to add a comment here stating that goreturns does both gofmt + goimports? I was a little confused initially